### DISCUSSION TITLE
retry on greylisting event

### DISCUSSION THREAD

**Discussion by z3d3m0n** [postalserver/postal#3229] [Help with using Postal]
Hello there

i have run a small test with sending to a mimecast domain and got a 451
451 Internal resource temporarily unavailable - https://community.mimecast.com/docs/DOC-1369#451 
postal showed a soft fail with Temporary SMTP delivery error when sending to 195.130.217.211:25 (eu-smtp-inbound-1.mimecast.com)

the 451 is a greylisting event usually a retry in 15-30-60 minutes is required
i there  feature to retry on the greylisting ?

thank you in advance

**Comment by willpower232** [comment]
> postal showed a soft fail

Soft fails are normally retried by Postal with increasing delay so it should be fine unless you've changed the setting for the maximum amount of retries.
